modules/persistent-storage-csi-migration-enable.adoc

```diff
@@ -8,7 +8,7 @@
 
 [NOTE]
 ====
-Enabling CSI automatic migration drains, and then restarts, all nodes in the cluster in sequence. This might take some time.
+Enabling Container Storage Interface (CSI) automatic migration drains, and then restarts, all nodes in the cluster in sequence. This might take some time.
```

Comment:
I would highlight the fact that this is section is only intended for testing purposes (testing CSI migration before being GA). For production system, the CSI migration will be automatically enabled and no manual operation is required (once the migration is GA). It's mentioned earlier in the doc but it would be best to avoid any confusion.